pkg/aws/tags/builder.go

```diff
@@ -48,6 +48,8 @@ func (p BuildParams) Build() map[string]string {
 		tags["Name"] = p.Name
 	}
 
+	tags[NameAWSProviderPrefix+p.ClusterName] = "owned"
+
 	// CAPA is also setting "cluster/$cluster_id=owned" tag which makes the
 	// resource to be managed by CAPA, so we skip that here.
```

Comment:
Guessing this comment is no longer relevant

Reply:
removed